packages/db/tests/test-executions.js

```diff
@@ -0,0 +1,56 @@
+const test = require('ava');
```

Comment:
Nit: can we move this test file to the `tests/translate` folder?

Reply:
Ah yeah, absolutely.